BigBatch/Views/BatchedVC1.swift

```diff
@@ -233,15 +222,27 @@ class BatchedVC1: UIViewController, UITableViewDelegate, UITableViewDataSource,
     }
     
     func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
-        
         return batchedCellDataArray.count
-        
-        
     }
     
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
         let cell = batchIngredientsTableView.dequeueReusableCell(withIdentifier: TotalBatchTableViewCell.batchCellIdentifier, for: indexPath) as! TotalBatchTableViewCell
-        let colorArray = [cocktail.blue, cocktail.red, cocktail.orange, cocktail.lightOrange, cocktail.forestGreen, cocktail.blue, cocktail.red, cocktail.orange, cocktail.lightOrange, cocktail.forestGreen, cocktail.blue, cocktail.red, cocktail.orange, cocktail.lightOrange, cocktail.forestGreen, cocktail.blue ]
+        let colorArray = [colorPalette.blue,
+                          colorPalette.red,
+                          colorPalette.orange,
+                          colorPalette.lightOrange,
+                          colorPalette.forestGreen,
+                          colorPalette.blue,
+                          colorPalette.red,
+                          colorPalette.orange,
+                          colorPalette.lightOrange,
+                          colorPalette.forestGreen,
+                          colorPalette.blue,
+                          colorPalette.red,
+                          colorPalette.orange,
+                          colorPalette.lightOrange,
+                          colorPalette.forestGreen,
+                          colorPalette.blue ]
         
```

Comment:
This would be cleaner if it was a method inside the ColorPalette struct you created. 
Put a method inside the ColorPalette struct that returns an array of Colors, then here you can just call

let ColorArray = ColorPalette.getColors()